dev-cli/container/Dockerfile

```diff
@@ -110,6 +110,31 @@ RUN cd /opt/aolibc && make CC="emcc -s WASM=1 -s MEMORY64=1 -s SUPPORT_LONGJMP=1
 ENV CC='emcc -s WASM=1'
 ENV NM='/emsdk/upstream/bin/llvm-nm'
 
+#####
+# Build gmp as a static library for WASM32
+#####
+COPY ./src/gmp-wasm /opt/gmp-wasm
+# ENV AR=emar
```

Comment:
Uncomment 117-121 and comment 122-123 if you prefer to build `libgmp.a` as part of the dockerfile build.